ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
+$icon-width: 24px;
+$icon-margin-right: 2rem;
+$question-padding: 1.5rem;
+
+.faq-block__icon {
+    font-size: 30px;
+    margin-right: $icon-margin-right;
+}
+
+.faq-block__question {
+    margin: 0;
+}
+
+.faq-block__text {
+    white-space: pre-wrap;
```

Comment:
do we need it? it prevents line breaking and we need it based on mockups
https://projects.invisionapp.com/d/main#/console/21368369/451746231/preview

Reply:
I feel we need it. 

![image](https://user-images.githubusercontent.com/7396837/122286159-3c12ac00-cef8-11eb-9b53-3752bbef7046.png)

![image](https://user-images.githubusercontent.com/7396837/122286481-93b11780-cef8-11eb-9922-8cd3cd46309c.png)